htdocs/wp-content/themes/hollandskroon/src/Providers/CommandServiceProvider.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace HK\Providers;
+
+use HK\Commands\PronamicIdealCleanupCommand;
+
+class CommandServiceProvider
```

Comment:
Ik zou je class geen ServiceProvider noemen als het niet de ServiceProvider van laravel extends.

Deze class lijkt ook wat overbodig. Je kunt de volgende regels in je function.php zetten: (en deze class verwijderen)

```
if (defined('WP_CLI') && WP_CLI) {
	\WP_CLI::add_command('pronamic:cleanup', PronamicIdealCleanupCommand::class);
}
```